nucliadb/nucliadb/common/cluster/settings.py

```diff
@@ -19,7 +19,10 @@
 #
 import enum
 
-from pydantic import BaseSettings, Field
+try:
+    from pydantic.v1 import BaseSettings, Field
+except ImportError:
+    from pydantic import BaseSettings, Field
```

Comment:
Since nucliadb is not meant to be used as a library, can't we assume that we will have a pydantic version `>=2.0.0` and simply have `from pydantic.v1 import BaseSettings, Field` ?

This doesn't apply to the libraries like:
- nucliadb_telemetry
- nucliadb_models
- nucliadb_utils
- nucliadb_sdk

For the sake of this contribution, if you want you can limit it to migrate anything under the `nucliadb/` folder and replacing pydantic-argparse dependency to your fork. What do you think?